src/frequenz/sdk/actor/power_distributing/_battery_status.py

```diff
@@ -62,8 +62,16 @@ class SetPowerResult:
 @dataclass
 class _ComponentStreamStatus:
     component_id: int
+    """Component id."""
+
+    data_recv_timer: Timer
+    """Timer that should be set when no component data was received for some time"""
```

Comment:
maybe `Timer that is set when no component data has been received for some time.` instead?

Reply:
Done